packages/desktop/src/app.tsx

```diff
@@ -1,30 +1,30 @@
 import { DataStory } from '@data-story/ui';
-import { Application, coreNodeProvider } from '@data-story/core';
+import { Application, coreNodeProvider, Diagram } from '@data-story/core';
 import { createRoot } from 'react-dom/client';
+import React from 'react';
+import { SaveComponent } from './save';
 // eslint-disable-next-line import/no-unresolved
 import '@data-story/ui/data-story.css';
 
 const container = document.getElementById('root');
 const root = createRoot(container);
 
-const app = new Application();
+export const App = ({ mode }: {mode?: 'js' | 'node'}) => {
+  const app = new Application()
+    .register(coreNodeProvider)
+    .boot();
 
-app.register([
-  coreNodeProvider
-]);
+  return (
+    <div style={{
+      height: '95vh',
+      margin: '0px',
+    }} data-cy="playground">
+      <DataStory
+        slotComponent={<SaveComponent  />}
+        server={{ type: 'JS', app }}
```

Comment:
Please change server type to SOCKET here. The `app` is defined as `dataStory` in desktop/src/index.ts